README.md

```diff
@@ -16,12 +16,79 @@ goose
 <a href="#open-source">Open Source</a>
 </p>
 
-`goose` assists in solving a wide range of programming and operational tasks. It is a live virtual developer you can interact with, guide, and learn from.
+**Welcome to `goose`, the Open-Source, Extensible AI Coding Assistant!** 
 
-To solve problems, `goose` breaks down instructions into sequences of tasks and carries them out using tools. Its ability to connect its changes with real outcomes (e.g. errors) and course correct is its most powerful and exciting feature. `goose` is free open source software and is built to be extensible and customizable.
+Unlike traditional coding tools, `goose` leverages advanced natural language processing to interpret complex commands into actionable, executable sequences. This allows for a coding assistant that not only writes code but also understands the intent behind your commands, enabling it to manage broader project scopes autonomously.
+
+```
+TODO: quick install instructions will go here in a callout
+```
+
+`TODO: slow video down and center it on the page`
 
 ![goose_demo](https://github.com/user-attachments/assets/0794eaba-97ab-40ef-af64-6fc7f68eb8e2)
 
+## What Sets Goose Apart?
+
+* **Adaptability**: `goose` excels in its ability to adapt to changes in code in real-time, making necessary adjustments as errors occur. This mimics a human developer's approach to coding, iteratively refining the work until perfection is achieved.
+* **Autonomy**: While tools like Copilot provide line-by-line code suggestions based on comments or partial code snippets, goose can take full command descriptions to manage entire projects with little to no ongoing input from developers.
+* **Customization and Extensibility**: Open-source and fully customizable, goose is built to support extensive user-defined modifications, unlike the mostly closed systems of Copilot and Cursor. Developers can tailor `goose` extensively to fit specific workflows, integrate with various toolkits, and enhance functionality through community contributions.
+* **Privacy and Security**: Operating locally on your device, `goose` ensures that your proprietary code and sensitive data are never sent to the cloud.
```

Comment:
They are sent to LLM's if you are using a provider like openai/anthropic though?

Reply:
**From Brad:**

> privacy and security: i'm not sure we do this that well because at the end of the day all the content does go to openai/anthropic etc?

Reply:
Hmm, I think what I was trying to accomplish here was to make the argument that Goose is all local / controlled by you. I don't know how compelling of an argument can be made but this is what i found in my research:

**cursor**

When I was looking at cursor it has its own backend that it uses to take your request and build the final prompt

Also cursor will index your codebase and upload it to their server to compute embeddings (though all plaintext code ceases to exist after the the life of the request)

The embeddings and metadata about the codebase is stored in their database 

**copilot**

Copilot does not appear to do any of the above -- the prompts and responses are discarded after use and they explicitly state it's not used for training data

But also, copilot will sort of 'rephrase' (if I understand correctly) parts of prompts and send them through Bing if copilot determines there's terms in the prompt where web grounding would improve the quality of the response. 

It's not shared with advertisers, and is de-identified / the prompt is not used verbatim